### PR TITLE
[C-2785] Update collection screen focus effect to fetch lineup

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Small add to update the collection screen to fetch the lineup metadata whenever the screen is focused

### Dragons

N/A

### How Has This Been Tested?

Manually tested

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

File: packages/web/src/common/store/pages/collection/sagas.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const { fetchCollection, fetchCollectionSucceeded, fetchCollectionFailed } =
 
 function* watchFetchCollection() {
   yield takeLatest(collectionActions.FETCH_COLLECTION, function* (action) {
-    const { id: collectionId, permalink } = action
+    const { id: collectionId, permalink, fetchLineup } = action
```
Comment (dylanjeffers) [inline packages/web/src/common/store/pages/collection/sagas.js:30]: oh interesting, i see! okay hmm could we not have a useEffect that just calls fetchLineupMetada like we have with desktop? i honestly like what youve done here though which just ensures fechLineup gets called in addition to fetchCollection :)

**Review by dylanjeffers — APPROVED** [review]
awesome addition

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/kj-Update-collection-screen-focus

File: packages/web/src/common/store/pages/collection/sagas.js (hunk 1 above)
Comment (Kyle-Shanks) [reply packages/web/src/common/store/pages/collection/sagas.js:30]: for some reason there was a prefix error so I did it this way. controls the flow this way which i think is what fixes the error

**Pull request merged**